Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -338,16 +339,14 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
 
     WebCore::IntPoint convertFromPluginToDocument(const WebCore::IntPoint&) const;
     WebCore::IntPoint convertFromDocumentToPlugin(const WebCore::IntPoint&) const;
-
+    WebCore::IntRect convertFromDocumentToPlugin(const WebCore::IntRect&) const;
+    std::optional<PDFDocumentLayout::PageIndex> pageIndexForDocumentPoint(const WebCore::IntPoint&) const;
```

Comment:
I prefer this down where it was, outside this grouping of coordinate conversion functions.

Reply:
Okay, I'll move it back.

Reply:
https://github.com/WebKit/WebKit/commit/8e6e24fe49a6d1813757c1a08554d2ab0f8841a6